pkg/cidata/cidata.TEMPLATE.d/user-data

```diff
@@ -33,7 +33,8 @@ users:
     lock_passwd: true
     ssh-authorized-keys:
     {{- range $val := .SSHPubKeys}}
-      - "{{$val}}"
+      - |
+        {{$val}}
```

Comment:
An alternative would be:

```suggestion
      - {{printf "%q" $val}}
```

but that too would require changes to `lima-init.sh`.

Reply:
This def seems more elegant - updated this PR as well as the other PR.

Reply:
It also has the advantage that the update to `lima-init.sh` is not _strictly_ required; it still works with the existing image; you just get additional backslashes in your `~/.ssh/authorized_keys` file, which is untidy, but they are just ignored, e.g.

```
lima-alpine:~$ cat .ssh/authorized_keys
ssh-ed25519 AAAA...MG \jan@host\
```

I've merged your change to alpine-lima and made a new release, but it is not urgent to bump it now.